src/plugin/file/kml/ui/kmlui.js

```diff
@@ -163,8 +163,15 @@ plugin.file.kml.ui.updatePlacemark = function(options) {
   // add the placemark to a parent if provided
   if (options['parent']) {
     var parent = plugin.file.kml.ui.verifyNodeInTree_(/** @type {!plugin.file.kml.ui.KMLNode} */ (options['parent']));
-    var cmd = new plugin.file.kml.cmd.KMLNodeAdd(placemark, parent);
-    os.commandStack.addCommand(cmd);
+    var currentParent = placemark.getParent();
+    if (currentParent) {
+      // update parent
+      currentParent.removeChild(placemark);
+      placemark.setParent(parent);
```

Comment:
Why does adding get a command on the stack but updating does not?

Reply:
I modified this to use the stack as-is, so removing and adding it. If we want to implement an update instead of just an add and remove that's fine, but might be better done outside of this ticket as it would affect more places in the code. Let me know what you think about the proposed changes.